app/login/form.tsx

```diff
@@ -14,22 +14,25 @@ export default function Form() {
         const response = await signIn('credentials', {
             email: formData.get('email'),
             password: formData.get('password'),
-            redirect: false,
+            redirect: true,
+            callbackUrl: '/'
         })
 
-   if (response?.error) {
-      {
-        setError("Invalid email/password.");
-      }
+        if (response?.error) {
+            {
+                setError("Invalid email/password.");
+            }
+        } else {
+            router.push('/');
```

Comment:
I'm pretty sure we don't need `router` anymore. The redirect should be handled by `signIn`.